### PR TITLE
Drizzlepac notebook debug and fix : Drizzle_wfpc2 and Sky_matching (PR 2 of 2)

### PR DESCRIPTION
**Pull request by dulude**
**Associated PRs**
- PR #78
- PR #128

**Important Note**
-  Due to revised security policies, pull requests can no longer  to be made from branches off of personal forks.
- This is the second transitional pull request meant to merge changes from a branch off a personal fork of hst_notebooks to a branch off of [hst_notebooks:main](https://github.com/spacetelescope/hst_notebooks). 
- PR #128 merged changes from a branch off a personal fork of hst_notebooks to a branch off of [hst_notebooks:main](https://github.com/spacetelescope/hst_notebooks) , [hst_notebooks:drizzlepacfix_kh](https://github.com/spacetelescope/hst_notebooks/tree/drizzlepacfix_kh).
- This second PR merges [hst_notebooks:drizzlepacfix_kh](https://github.com/spacetelescope/hst_notebooks/tree/drizzlepacfix_kh) into [hst_notebooks:main](https://github.com/spacetelescope/hst_notebooks) 

**Summary**
Drizzlepac notebook fix to two notebooks: Drizzle_wfpc2 and sky_matching.

Drizzle_wfpc2: resolved "CalledProcessError".
fixed a couple of syntax errors that stems from new updates to AstroDrizzle.
Added a section that made the DQ array for the simulated fits, because Astrodrizzle could not run the simulated files without a DQ array.

Sky_Matching:

Tweakreg cell: current parameters had trouble finding stars to match. I changed the parameters and added a couple to help find matches between images.

tweakback cell:
changed drz_files; was calling the single files too, should only call the *0_drz.files. Also changed the input as the input has to be f + ‘[sci,1]’ to grab the header from the sci header
changed tweakback.tweakback to tweakback.apply_tweak, and added the correct parameters for it to work.

----------------
This notebook checklist has been made available to us by the [Notebooks For All team](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/event-hackathon/notebook-authoring-checklist.md).
Its purpose is to serve as a guide for both the notebook author and the technical reviewer highlighting critical aspects to consider when striving to develop an accessible and effective notebook.

### The First Cell

- [ ] The title of the notebook in a first-level heading (eg. `<h1>` or `# in markdown`).
- [ ] A brief description of the notebook.
- [ ] A table of contents in an [ordered list](https://www.markdownguide.org/basic-syntax/#ordered-lists) (`1., 2.,` etc. in Markdown).
- [ ] The author(s) and affiliation(s) (if relevant).
- [ ] The date first published.
- [ ] The date last edited (if relevant).
- [ ] A link to the notebook's source(s) (if relevant).

### The Rest of the Cells

- [ ] There is only one H1 (`#` in Markdown) used in the notebook.
- [ ] The notebook uses other [heading tags](https://www.markdownguide.org/basic-syntax/#headings) in order (meaning it does not skip numbers). 

## Text

- [ ] All link text is descriptive. It tells users where they will be taken if they open the link.
- [ ] All acronyms are defined at least the first time they are used. 
- [ ] Field-specific/specialized terms are used when needed, but not excessively.

## Code

- [ ] Code sections are introduced and explained before they appear in the notebook. This can be fulfilled with a heading in a prior Markdown cell, a sentence preceding it, or a code comment in the code section.
- [ ] Code has explanatory comments (if relevant). This is most important for long sections of code.
- [ ] If the author has control over the syntax highlighting theme in the notebook, that theme has enough color contrast to be legible.
- [ ] Code and code explanations focus on one task at a time. Unless comparison is the point of the notebook, only one method for completing the task is described at a time.

### Images

- [ ] All images (jpg, png, svgs) have an [image description](https://www.w3.org/WAI/tutorials/images/decision-tree/). This could be
    - [ ] [Alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt) (an `alt` property) 
    - [ ] [Empty alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#decorative_images) for decorative images/images meant to be skipped (an `alt` attribute with no value)
    - [ ] Captions
    - [ ] If no other options will work, the image is decribed in surrounding paragraphs.

- [ ] Any [text present in images](https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html) exists in a text form outside of the image (this can be alt text, captions, or surrounding text.)

### Visualizations

- [ ] All visualizations have an image description. Review the previous section, Images, for more information on how to add it.
- [ ] [Visualization descriptions](http://diagramcenter.org/specific-guidelines-e.html) include
    - [ ] The type of visualization (like bar chart, scatter plot, etc.)
    - [ ] Title
    - [ ] Axis labels and range
    - [ ] Key or legend
    - [ ] An explanation of the visualization's significance to the notebook (like the trend, an outlier in the data, what the author learned from it, etc.)

- [ ] All visualizations and their parts have [enough color contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) ([color contrast checker](https://webaim.org/resources/contrastchecker/)) to be legible. Remember that transparent colors have lower contrast than their opaque versions.
- [ ] All visualizations [convey information with more visual cues than color coding](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html). Use text labels, patterns, or icons alongside color to achieve this.
- [ ] All visualizations have an additional way for notebook readers to access the information. Linking to the original data, including a table of the data in the same notebook, or sonifying the plot are all options.

### CODE REVIEW

**Comment by kyhuynh108** [comment]
@haticekaratay @camipacifici these two drizzlepac notebooks are ready for final review. Please let me know if theres anything else that needs to be done for these notebooks. Thanks

File: notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb [hunk 1]
```diff
@@ -18,7 +18,28 @@
    "cell_type": "markdown",
```
Comment (haticekaratay) [inline notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb:18]: <p>Thanks for your work! I wanted to bring your attention to a small detail: there seems to be an issue with the prior WFPCS <code>example</code>&nbsp;link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (haticekaratay) [inline notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb:18]: <p>Also, there seems to be an issue with the link to <code>'Making WFPC2 Images Compatible with AstroDrizzle'</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (haticekaratay) [inline notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb:18]: <p>Also an unaccessible link to Appendix B.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Comment by haticekaratay** [comment]
Thanks for your work, @kyhuynh108. I updated the sky_matching notebook to minimize the back-and-forth iterations. I've also left some comments in the drizzle_wfc2 for you to update.

**Comment by kyhuynh108** [comment]
@haticekaratay All of the URLs are fixed. However, I believe a newer version of the drizzlepac handbook is under progress so some URLs may be affected when that comes out. Thanks

**Review by haticekaratay — APPROVED** [review]
Thanks for the updates. All looks good to me!

**Pull request merged**